plenum/test/validator_info/test_validator_info.py

```diff
@@ -127,6 +127,10 @@ def test_node_info_section(info, node):
     assert len(info['Node_info']['Catchup_status']['Last_txn_3PC_keys']) == 3
     assert info['Node_info']['Catchup_status']['Ledger_statuses']
     assert len(info['Node_info']['Catchup_status']['Ledger_statuses']) == 3
+    for idx, ls in info['Node_info']['Catchup_status']['Ledger_statuses'].items():
```

Comment:
I'm not sure that having information about freshness in 'Catchup_status' is a good idea. Maybe we should create a separate section for this? For example, `Freshness_status`?